cves/kernel/CVE-2012-6545.yml

```diff
@@ -401,37 +412,37 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: false
     note:
   least_privilege:
-    applies:
+    applies: false
     note:
   frameworks_are_optional:
-    applies:
+    applies: false
     note:
   native_wrappers:
-    applies:
+    applies: false
     note:
   distrust_input:
-    applies:
+    applies: false
```

Comment:
The code fix initializes the struct with memset(0) before filling it, which can be seen as a form of input validation to avoid potential leaks

Reply:
In a way, but I felt it was too indirect of input.